futures-channel/src/mpsc/mod.rs

```diff
@@ -319,34 +321,28 @@ impl SenderTask {
     }
 }
 
-/// Creates an in-memory channel implementation of the `Stream` trait with
-/// bounded capacity.
+/// Creates a bounded mpsc channel for communicating between asynchronous tasks.
 ///
-/// This method creates a concrete implementation of the `Stream` trait which
-/// can be used to send values across threads in a streaming fashion. This
-/// channel is unique in that it implements back pressure to ensure that the
-/// sender never outpaces the receiver. The channel capacity is equal to
-/// `buffer + num-senders`. In other words, each sender gets a guaranteed slot
-/// in the channel capacity, and on top of that there are `buffer` "first come,
-/// first serve" slots available to all senders.
+/// Being bounded, this channel provides back pressure to ensure that the sender
+/// outpaces the receiver by only a limited amount. The channel's capacity is
+/// equal to `buffer + num-senders`. In other words, each sender gets a
+/// guaranteed slot in the channel capacity, and on top of that there are
+/// `buffer` "first come, first serve" slots available to all senders.
 ///
-/// The `Receiver` returned implements the `Stream` trait and has access to any
-/// number of the associated combinators for transforming the result.
+/// The [`Receiver`](Receiver) returned implements the [`Stream`](Stream) trait,
+/// while [`Sender`](Sender) implements [`Sink`](Sink).
```

Comment:
The link to `Sink` here will fail, unfortunately, because `futures_channel` doesn't depend on `futures_sink`.